client/examples/local_serving.py

```diff
@@ -1,15 +1,15 @@
 import featureform as ff
 
-client = ff.ServingClient(local=True)
+client = ff.Client(local=True)
 
-training_set = client.training_set("fraud_training", "quickstart")
+training_set = client.training_set("fraud_training", "courageous_jones")
```

Comment:
the variant would change each time you run the `local_quickstart.py`? should we hardcode the variant?

Reply:
This is for my own testing, will change later before we merge